docs/flow-results-connector.md

```diff
@@ -0,0 +1,112 @@
+## FLOW RESULTS PACKAGE CONNECTOR
+
+This connector is designed in Apache NiFi to automate the process of transferring flow-results packaged data to a table in a Postgres database or store AVRO formatted response results on disk. The connector ingests data in three different ways:
+
+1. __Flow results API endpoint__
+The schema for the table is derived from the descriptor of a package retrieved from `/flow-results/packages/[form_uuid]`. Once the schema for the table is generated, a table is created in the database. Data is then retrieved from the response endpoint: `/flow-results/packages/[form_uuid]/responses`. The connector requires a JSON configuration with the keys `form_uuid`, `base_url` and optionally, the `table_name` the user desires the data to be stored in. The JSON config can is formatted as below:
+
+```
+{
+  "form_uuid": "[package form UUID]",
+  "table_name": "[preferred Postgres table name]",
+  "base_url": "[Base URL to get data from]"	
+}
+```
+
+
+> Note on API authentication
+> For APIs that require authentication, edit processors that make GET requests to the API for survey and response named `GET flow results survey from API` and `GET flow results responses from API`. Enter the Basic Authentication Username and Basic Authentication Password or add a property for Authorization token by clicking + in properties.
+
+
+
+2. __Survey and response JSON feed__
+For JSON feed, under `Flow results JSON` processor, add the survey JSON to survey attribute and response JSON to the response attribute and custom text property.
+
+
+
+3. __Pushed submissions__
```

Comment:
Can we rename "Pushed submissions" to "Publish responses" to be consistent with [API specifications](https://github.com/FLOIP/flow-results/blob/master/api-specification.md#publish-responses-to-a-package)?